src/main/java/rasputin/command/ExitCommand.java

```diff
@@ -0,0 +1,46 @@
+package rasputin.command;
+
+import javafx.application.Platform;
+import rasputin.task.TaskList;
+
+import rasputin.storage.Storage;
+
+import rasputin.gui.Ui;
```

Comment:
Perhaps the rasputin imports can be put together, while the javafx import can be put on top for better readability?